Missing repository information in pom.xml for org.dstovall:onejar-maven-plugin

pom.xml is missing repository information for onejar-maven-plugin:

```
[ERROR] BUILD ERROR
[INFO] ------------------------------------------------------------------------
[INFO] Error building POM (may not be this project's POM).


Project ID: org.dstovall:onejar-maven-plugin

Reason: POM 'org.dstovall:onejar-maven-plugin' not found in repository: Unable to download the artifact from any repository

  org.dstovall:onejar-maven-plugin:pom:1.4.5

from the specified remote repositories:
  central (http://repo1.maven.org/maven2)

 for project org.dstovall:onejar-maven-plugin
```

--- a/pom.xml
+++ b/pom.xml
@@ -117,4 +117,10 @@
             </plugin>
         </plugins>
     </build>
+    <pluginRepositories>
+        <pluginRepository>
+            <id>onejar-maven-plugin.googlecode.com</id>
+            <url>http://onejar-maven-plugin.googlecode.com/svn/mavenrepo</url>
+        </pluginRepository>
+    </pluginRepositories>
 </project>

